<!doctype html>
<html class="docs-version-current" lang="en" dir="ltr">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width,initial-scale=1">
<meta name="generator" content="Docusaurus v0.0.0-4406">
<link rel="alternate" type="application/rss+xml" href="/NeuroNotes/blog/rss.xml" title="NeuroNotes RSS Feed">
<link rel="alternate" type="application/atom+xml" href="/NeuroNotes/blog/atom.xml" title="NeuroNotes Atom Feed"><title data-react-helmet="true">2012 Sequencing the Connectome | NeuroNotes</title><meta data-react-helmet="true" name="twitter:card" content="summary_large_image"><meta data-react-helmet="true" property="og:url" content="https://tesla-cat.github.io/NeuroNotes/docs/Sequencing the Connectome/2012 Sequencing the Connectome"><meta data-react-helmet="true" name="docusaurus_locale" content="en"><meta data-react-helmet="true" name="docusaurus_version" content="current"><meta data-react-helmet="true" name="docusaurus_tag" content="docs-default-current"><meta data-react-helmet="true" property="og:title" content="2012 Sequencing the Connectome | NeuroNotes"><meta data-react-helmet="true" name="description" content="- Paper"><meta data-react-helmet="true" property="og:description" content="- Paper"><link data-react-helmet="true" rel="icon" href="/NeuroNotes/img/favicon.ico"><link data-react-helmet="true" rel="canonical" href="https://tesla-cat.github.io/NeuroNotes/docs/Sequencing the Connectome/2012 Sequencing the Connectome"><link data-react-helmet="true" rel="alternate" href="https://tesla-cat.github.io/NeuroNotes/docs/Sequencing the Connectome/2012 Sequencing the Connectome" hreflang="en"><link data-react-helmet="true" rel="alternate" href="https://tesla-cat.github.io/NeuroNotes/docs/Sequencing the Connectome/2012 Sequencing the Connectome" hreflang="x-default"><link rel="stylesheet" href="/NeuroNotes/assets/css/styles.9fbd5245.css">
<link rel="preload" href="/NeuroNotes/assets/js/runtime~main.9fa0cc15.js" as="script">
<link rel="preload" href="/NeuroNotes/assets/js/main.288aa2cb.js" as="script">
</head>
<body>
<script>!function(){function t(t){document.documentElement.setAttribute("data-theme",t)}var e=function(){var t=null;try{t=localStorage.getItem("theme")}catch(t){}return t}();t(null!==e?e:"light")}()</script><div id="__docusaurus">
<div role="region"><a href="#" class="skipToContent_OuoZ">Skip to main content</a></div><nav class="navbar navbar--fixed-top"><div class="navbar__inner"><div class="navbar__items"><button aria-label="Navigation bar toggle" class="navbar__toggle clean-btn" type="button" tabindex="0"><svg width="30" height="30" viewBox="0 0 30 30" aria-hidden="true"><path stroke="currentColor" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2" d="M4 7h22M4 15h22M4 23h22"></path></svg></button><a class="navbar__brand" href="/NeuroNotes/"><div class="navbar__logo"><img src="/NeuroNotes/img/logo.svg" alt="NeuroNotes Logo" class="themedImage_TMUO themedImage--light_4Vu1"><img src="/NeuroNotes/img/logo.svg" alt="NeuroNotes Logo" class="themedImage_TMUO themedImage--dark_uzRr"></div><b class="navbar__title">NeuroNotes</b></a><a class="navbar__item navbar__link navbar__link--active" href="/NeuroNotes/docs/Sequencing the Connectome/Papers">Notes</a></div><div class="navbar__items navbar__items--right"><div class="toggle_iYfV toggle_2i4l toggleDisabled_xj38"><div class="toggleTrack_t-f2" role="button" tabindex="-1"><div class="toggleTrackCheck_mk7D"><span class="toggleIcon_pHJ9">ğŸŒœ</span></div><div class="toggleTrackX_dm8H"><span class="toggleIcon_pHJ9">ğŸŒ</span></div><div class="toggleTrackThumb_W6To"></div></div><input type="checkbox" class="toggleScreenReader_h9qa" aria-label="Switch between dark and light mode"></div></div></div><div role="presentation" class="navbar-sidebar__backdrop"></div></nav><div class="main-wrapper docs-wrapper docs-doc-page"><div class="docPage_lDyR"><button aria-label="Scroll back to top" class="clean-btn theme-back-to-top-button backToTopButton_i9tI" type="button"></button><aside class="docSidebarContainer_0YBq"><div class="sidebar_a3j0"><nav class="menu thin-scrollbar menu_cyFh"><ul class="theme-doc-sidebar-menu menu__list"><li class="theme-doc-sidebar-item-category theme-doc-sidebar-item-category-level-1 menu__list-item"><div class="menu__list-item-collapsible"><a class="menu__link menu__link--sublist menu__link--active hasHref_TwRn" href="/NeuroNotes/docs/Sequencing the Connectome/Papers">Sequencing the Connectome</a></div><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-2 menu__list-item"><a class="menu__link" tabindex="0" href="/NeuroNotes/docs/Sequencing the Connectome/Papers">Papers</a></li><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-2 menu__list-item"><a class="menu__link menu__link--active" aria-current="page" tabindex="0" href="/NeuroNotes/docs/Sequencing the Connectome/2012 Sequencing the Connectome">2012 Sequencing the Connectome</a></li><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-2 menu__list-item"><a class="menu__link" tabindex="0" href="/NeuroNotes/docs/Sequencing the Connectome/2014 In vivo generation of DNA sequence diversity for cellular barcoding">2014 In vivo generation of DNA sequence diversity for cellular barcoding</a></li><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-2 menu__list-item"><a class="menu__link" tabindex="0" href="/NeuroNotes/docs/Sequencing the Connectome/2016 High-throughput mapping of single neuron projections by sequencing of barcoded RNA">2016 High-throughput mapping of single neuron projections by sequencing of barcoded RNA</a></li><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-2 menu__list-item"><a class="menu__link" tabindex="0" href="/NeuroNotes/docs/Sequencing the Connectome/2017 Using high-throughput barcode sequencing to efficiently map connectomes">2017 Using high-throughput barcode sequencing to efficiently map connectomes</a></li><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-2 menu__list-item"><a class="menu__link" tabindex="0" href="/NeuroNotes/docs/Sequencing the Connectome/2019 Network cloning using DNA barcodes">2019 Network cloning using DNA barcodes</a></li></ul></li><li class="theme-doc-sidebar-item-category theme-doc-sidebar-item-category-level-1 menu__list-item menu__list-item--collapsed"><div class="menu__list-item-collapsible"><a class="menu__link menu__link--sublist hasHref_TwRn" href="/NeuroNotes/docs/fMRI/Papers">fMRI</a></div></li><li class="theme-doc-sidebar-item-category theme-doc-sidebar-item-category-level-1 menu__list-item menu__list-item--collapsed"><div class="menu__list-item-collapsible"><a class="menu__link menu__link--sublist hasHref_TwRn" href="/NeuroNotes/docs/Neuromorphic/2021 Neuromorphic electronics based on copying and pasting the brain">Neuromorphic</a></div></li><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-1 menu__list-item"><a class="menu__link" href="/NeuroNotes/docs/Links">Links</a></li></ul></nav></div></aside><main class="docMainContainer_r8cw"><div class="container padding-top--md padding-bottom--lg"><div class="row"><div class="col docItemCol_zHA2"><div class="docItemContainer_oiyr"><article><div class="tocCollapsible_aw-L theme-doc-toc-mobile tocMobile_Tx6Y"><button type="button" class="clean-btn tocCollapsibleButton_zr6a">On this page</button></div><div class="theme-doc-markdown markdown"><header><h1>2012 Sequencing the Connectome</h1></header><ul><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3479097/" target="_blank" rel="noopener noreferrer">Paper</a></li><li>BOINC: barcoding of individual neuronal connections</li></ul><h3 class="anchor anchorWithStickyNavbar_y2LR" id="prerequisites">prerequisites<a class="hash-link" href="#prerequisites" title="Direct link to heading">â€‹</a></h3><ul><li><strong>excision (åˆ‡é™¤)</strong>: The removal of a section of double-stranded DNA that is faulty due to mutation or incorrect replication</li><li><strong>inversion (æŸ“è‰²ä½“å€’ä½)</strong>: chromosome rearrangement in which a segment of a chromosome is reversed end-to-end</li><li><strong>ç¯åŒ–é‡ç»„é…¶ (cyclization recombinaseï¼ŒCre)</strong>: å¯åœ¨æ— ä»»ä½•è¾…åŠ©å› å­çš„ä½œç”¨ä¸‹ä¸ä¸åŒå½¢å¼çš„loxPï¼ˆlocus X over P1ï¼‰DNAåºåˆ—ç›´æ¥ç»“åˆå¹¶ç›¸äº’ä½œç”¨ï¼Œä»è€Œå¯åœ¨ç‰¹å®šé¶ç‚¹è¿›è¡Œç‰¹å®šåŸºå› çš„æ•²å…¥ã€æ•²é™¤ç­‰ï¼Œå…·æœ‰ä½œç”¨åŸç†ç®€å•ã€ç©ºé—´ç‰¹å¼‚æ€§é«˜ã€é‡ç»„æ•ˆç‡é«˜ç­‰ä¼˜ç‚¹è€Œè¢«å¹¿æ³›åº”ç”¨ã€‚</li></ul><h3 class="anchor anchorWithStickyNavbar_y2LR" id="why-is-the-single-neuron-connectome-important">Why Is the Single Neuron Connectome Important?<a class="hash-link" href="#why-is-the-single-neuron-connectome-important" title="Direct link to heading">â€‹</a></h3><div class="ctr"><img src="/NeuroNotes/img/p1.png" width="300"></div><ul><li><p><code>motion detection circuit</code> (top): </p><ul><li>sequential activation of input neurons from left to right (1, 4, 7, 10) will generate less activity in the output neuron (0) than activation from right to left (10, 7, 4, 1)</li></ul></li><li><p><code>lateral inhibition circuit</code> (bottom): </p><ul><li>wired similarly, but the addition of a few extra inhibitory connections renders it insensitive to directional motion </li></ul></li><li><p>C. elegans:</p><ul><li>302 neurons connected by about 7,000 synapses. </li><li>Interestingly, the utility of the connectome in C. elegans is somewhat limited because function is highly multiplexed, with different neurons performing different roles depending on the state of neuromodulation, possibly as a mechanism for compensating for the small number of neurons</li></ul></li></ul><h3 class="anchor anchorWithStickyNavbar_y2LR" id="current-approaches-to-the-connectome">Current Approaches to the Connectome<a class="hash-link" href="#current-approaches-to-the-connectome" title="Direct link to heading">â€‹</a></h3><ul><li>physiology<ul><li>Callaway and colleagues used <code>laser scanning photostimulation</code> to probe connectivity in visual cortex. They found that if two nearby neurons in layer 2/3 are connected, then they share input from single neurons in layer 4, but if they are not connected they do not share input. Thus, the input from layer 4 to layer 2/3 appears to consist of at least two independent &quot;subnetworks&quot;, which happen to overlap in space. </li><li>Chklovskii and colleagues used whole cell methods to assess connectivity among triplets of neurons. By enumerating all 16 possible ways that three neurons can be connected, they discovered that several connectivity motifs were overrepresented above the chance levels predicted by the pairwise connection probabilities. Thus connectivity among triplets of cortical neurons deviates markedly from the null hypothesis of random connectivity. </li></ul></li><li>light microscopy<ul><li><code>GRASP</code> (GFP Reconstituted Across Synaptic Partners) allows synaptic contacts to be resolved at the level of light microscopy. </li><li><code>Brainbow</code> can be used to trace axons and dendrites over considerable distance. This technique relies on stochastic and combinatorial expression of several fluorophores (XFPs). Each neuron expresses a random collection of up to four different XFPs in different ratios, to achieve a theoretical palette of more than 100 different colors. The randomization is achieved by clever application of <code>Cre-lox recombination</code>, wherein the protein Cre recombinase catalyzes the inversion or excision of DNA between a pair of short (34 nucleotide) sequences termed lox sites. </li><li>Finally, it is now possible to image an entire mouse brain using <code>two-photon microscopy</code> in hours or days. </li></ul></li><li>electron microscopy (EM)<ul><li>EM is required because light microscopy does not have sufficient resolution to establish whether two nearby neuronal processes are merely close or whether they have actually formed a synapse. </li><li>Reconstruction of serial electron micrographs has yielded what to date is the only complete connectome, that of C. elegans. However, even for this simple nervous system, the reconstruction required a heroic effort -- over 50 person-years of labor to collect and analyze the images.</li></ul></li></ul><h3 class="anchor anchorWithStickyNavbar_y2LR" id="dna-sequencing-as-a-novel-method-of-solving-the-connectome">DNA Sequencing as a Novel Method of Solving the Connectome<a class="hash-link" href="#dna-sequencing-as-a-novel-method-of-solving-the-connectome" title="Direct link to heading">â€‹</a></h3><ul><li>BOINC: three components</li></ul><div class="ctr"><img src="/NeuroNotes/img/p4.png" width="300"></div><h4 class="anchor anchorWithStickyNavbar_y2LR" id="first-each-neuron-must-be-labeled-with-a-unique-sequence-of-nucleotidesa-dna-barcode">First, each neuron must be labeled with a unique sequence of nucleotidesâ€”a DNA &quot;barcode&quot;.<a class="hash-link" href="#first-each-neuron-must-be-labeled-with-a-unique-sequence-of-nucleotidesa-dna-barcode" title="Direct link to heading">â€‹</a></h4><ul><li>The requisite barcoding is conceptually similarâ€”though different in detailâ€”to the generation of antibody diversity by B cells in the immune system through somatic recombination (ä½“ç»†èƒé‡ç»„). </li><li>The idea of barcoding individual neurons is inspired by <code>Brainbow</code>, except that here DNA sequences substitute for fluorophores (XFPs). The advantage of using sequences is diversity: whereas Brainbow allows for at most hundreds of color combinations, a barcode consisting of even 20 random nucleotides can uniquely label 4^20 neurons, far more than the number of neurons (&lt;10^8) in a mouse brain.</li><li>One strategy for generating sufficient diversity to barcode every neuron&#x27;s DNA uniquely is shown below. In this strategy, inspired by Brainbow, each cell&#x27;s genome contains a cassette consisting of a sequence of short unique barcode elements Aâ€¦Eâ€¦ (top). Each barcode element is flanked by recombination sites (triangles). Upon expression of a suitable recombinase (é‡ç»„é…¶), these barcode elements shuffle and invert (shown here by inverted letter). The theoretical diversity that can be generated by this is 2^N<!-- -->*<!-- -->N!, where N is the number of barcode elements. For a cassette containing N=12 elements, the theoretical diversity is 2e12, far more than needed to barcode the 1e8 neurons in a mouse brain uniquely. Note that if a conventional recombinase like <code>cre</code> or <code>flp</code> is used here, <code>excision</code> will dominate over <code>inversion</code> and the resulting diversity scales with the number of barcode elements N. To avoid excision we use <code>RCI</code>, a recombinase that inverts but does <strong>NOT</strong> excise. (typo in paper)</li></ul><div class="ctr"><img src="/NeuroNotes/img/p2.png" width="400"></div><h4 class="anchor anchorWithStickyNavbar_y2LR" id="second-barcodes-from-synaptically-connected-neurons-must-be-associated">Second, barcodes from synaptically connected neurons must be associated.<a class="hash-link" href="#second-barcodes-from-synaptically-connected-neurons-must-be-associated" title="Direct link to heading">â€‹</a></h4><ul><li>One way to associate a pre- and postsynaptic barcode is by means of a <code>transsynaptic virus</code> such as <code>rabies (ç‹‚çŠ¬ç—…)</code> or <code>pseudorabies (PRV) (ä¼ªç‹‚çŠ¬ç—…)</code>. These viruses have evolved exquisite mechanisms for moving genetic material across synapses and have been used extensively for tracing neural circuits in rodents. To share barcodes across synapses, the virus must be engineered to carry the barcode within its own genetic sequence. After transsynaptic spread of the virus each postsynaptic neuron can be thought of as a &quot;bag of barcodes&quot;, consisting of copies of its own &quot;host&quot; barcodes, along with &quot;invader&quot; barcodes from presynaptically coupled neurons.</li></ul><h4 class="anchor anchorWithStickyNavbar_y2LR" id="finally-barcodes-from-synaptically-connected-neurons-must-be-joined-into-single-pieces-of-dna-for-high-throughput-sequencing">Finally, barcodes from synaptically connected neurons must be joined into single pieces of DNA for high-throughput sequencing.<a class="hash-link" href="#finally-barcodes-from-synaptically-connected-neurons-must-be-joined-into-single-pieces-of-dna-for-high-throughput-sequencing" title="Direct link to heading">â€‹</a></h4><ul><li>Barcodes are joined in vivo (æ´»ä½“), so there is no need to isolate individual neurons prior to extracting DNA. Since only those pairs associated in vivo are actually joined, observing a host-invader barcode pair indicates that the host and the invader were synaptically coupled. </li><li>One strategy for joining barcodes is based on <code>phiC31</code> integrase (æ•´åˆé…¶). PhiC31 mediates the integration of a <code>35-nucleotide AttB</code> site with a <code>35-nucleotide AttP</code> site to form an <code>AttL</code> and an <code>AttR</code> site. Because the AttL and AttR sites are not targets of phiC31, this reaction is irreversible (unlike comparable reactions with cre and flp). Once the barcodes are joined, they can be amplified by PCR (using primers complementary to the arrows) for sequencing.</li></ul><div class="ctr"><img src="/NeuroNotes/img/p3.png" width="300"></div><p>We are developing an approach based on <code>PRV amplicons</code>. Although there are many technical problems, including PRV toxicity and monosynaptic spread, which need to be addressed, this approach promises to offer a proof of principle for our proposal to convert connectivity into a sequencing problem.</p></div><footer class="theme-doc-footer docusaurus-mt-lg"><div class="theme-doc-footer-edit-meta-row row"><div class="col"><a href="https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/docs/1-Sequencing the Connectome/2012 Sequencing the Connectome.md" target="_blank" rel="noreferrer noopener" class="theme-edit-this-page"><svg fill="currentColor" height="20" width="20" viewBox="0 0 40 40" class="iconEdit_mS5F" aria-hidden="true"><g><path d="m34.5 11.7l-3 3.1-6.3-6.3 3.1-3q0.5-0.5 1.2-0.5t1.1 0.5l3.9 3.9q0.5 0.4 0.5 1.1t-0.5 1.2z m-29.5 17.1l18.4-18.5 6.3 6.3-18.4 18.4h-6.3v-6.2z"></path></g></svg>Edit this page</a></div><div class="col lastUpdated_mt2f"></div></div></footer></article><nav class="pagination-nav docusaurus-mt-lg" aria-label="Docs pages navigation"><div class="pagination-nav__item"><a class="pagination-nav__link" href="/NeuroNotes/docs/Sequencing the Connectome/Papers"><div class="pagination-nav__sublabel">Previous</div><div class="pagination-nav__label">Papers</div></a></div><div class="pagination-nav__item pagination-nav__item--next"><a class="pagination-nav__link" href="/NeuroNotes/docs/Sequencing the Connectome/2014 In vivo generation of DNA sequence diversity for cellular barcoding"><div class="pagination-nav__sublabel">Next</div><div class="pagination-nav__label">2014 In vivo generation of DNA sequence diversity for cellular barcoding</div></a></div></nav></div></div><div class="col col--3"><div class="tableOfContents_vrFS thin-scrollbar theme-doc-toc-desktop"><ul class="table-of-contents table-of-contents__left-border"><li><a href="#prerequisites" class="table-of-contents__link toc-highlight">prerequisites</a></li><li><a href="#why-is-the-single-neuron-connectome-important" class="table-of-contents__link toc-highlight">Why Is the Single Neuron Connectome Important?</a></li><li><a href="#current-approaches-to-the-connectome" class="table-of-contents__link toc-highlight">Current Approaches to the Connectome</a></li><li><a href="#dna-sequencing-as-a-novel-method-of-solving-the-connectome" class="table-of-contents__link toc-highlight">DNA Sequencing as a Novel Method of Solving the Connectome</a></li></ul></div></div></div></div></main></div></div><footer class="footer footer--dark"><div class="container container-fluid"><div class="footer__bottom text--center"><div class="footer__copyright">Copyright Â© 2022 NeuroNotes, Inc. Built with Docusaurus.</div></div></div></footer></div>
<script src="/NeuroNotes/assets/js/runtime~main.9fa0cc15.js"></script>
<script src="/NeuroNotes/assets/js/main.288aa2cb.js"></script>
</body>
</html>