<!doctype html>
<html class="docs-version-current" lang="en" dir="ltr">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width,initial-scale=1">
<meta name="generator" content="Docusaurus v0.0.0-4406">
<link rel="alternate" type="application/rss+xml" href="/NeuroNotes/blog/rss.xml" title="NeuroNotes RSS Feed">
<link rel="alternate" type="application/atom+xml" href="/NeuroNotes/blog/atom.xml" title="NeuroNotes Atom Feed"><title data-react-helmet="true">2017 Using high-throughput barcode sequencing to efficiently map connectomes | NeuroNotes</title><meta data-react-helmet="true" name="twitter:card" content="summary_large_image"><meta data-react-helmet="true" property="og:url" content="https://tesla-cat.github.io/NeuroNotes/docs/Sequencing the Connectome/2017 Using high-throughput barcode sequencing to efficiently map connectomes"><meta data-react-helmet="true" name="docusaurus_locale" content="en"><meta data-react-helmet="true" name="docusaurus_version" content="current"><meta data-react-helmet="true" name="docusaurus_tag" content="docs-default-current"><meta data-react-helmet="true" property="og:title" content="2017 Using high-throughput barcode sequencing to efficiently map connectomes | NeuroNotes"><meta data-react-helmet="true" name="description" content="- Paper"><meta data-react-helmet="true" property="og:description" content="- Paper"><link data-react-helmet="true" rel="icon" href="/NeuroNotes/img/favicon.ico"><link data-react-helmet="true" rel="canonical" href="https://tesla-cat.github.io/NeuroNotes/docs/Sequencing the Connectome/2017 Using high-throughput barcode sequencing to efficiently map connectomes"><link data-react-helmet="true" rel="alternate" href="https://tesla-cat.github.io/NeuroNotes/docs/Sequencing the Connectome/2017 Using high-throughput barcode sequencing to efficiently map connectomes" hreflang="en"><link data-react-helmet="true" rel="alternate" href="https://tesla-cat.github.io/NeuroNotes/docs/Sequencing the Connectome/2017 Using high-throughput barcode sequencing to efficiently map connectomes" hreflang="x-default"><link rel="stylesheet" href="/NeuroNotes/assets/css/styles.9fbd5245.css">
<link rel="preload" href="/NeuroNotes/assets/js/runtime~main.1c73bbed.js" as="script">
<link rel="preload" href="/NeuroNotes/assets/js/main.31e959ff.js" as="script">
</head>
<body>
<script>!function(){function t(t){document.documentElement.setAttribute("data-theme",t)}var e=function(){var t=null;try{t=localStorage.getItem("theme")}catch(t){}return t}();t(null!==e?e:"light")}()</script><div id="__docusaurus">
<div role="region"><a href="#" class="skipToContent_OuoZ">Skip to main content</a></div><nav class="navbar navbar--fixed-top"><div class="navbar__inner"><div class="navbar__items"><button aria-label="Navigation bar toggle" class="navbar__toggle clean-btn" type="button" tabindex="0"><svg width="30" height="30" viewBox="0 0 30 30" aria-hidden="true"><path stroke="currentColor" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2" d="M4 7h22M4 15h22M4 23h22"></path></svg></button><a class="navbar__brand" href="/NeuroNotes/"><div class="navbar__logo"><img src="/NeuroNotes/img/logo.svg" alt="NeuroNotes Logo" class="themedImage_TMUO themedImage--light_4Vu1"><img src="/NeuroNotes/img/logo.svg" alt="NeuroNotes Logo" class="themedImage_TMUO themedImage--dark_uzRr"></div><b class="navbar__title">NeuroNotes</b></a><a class="navbar__item navbar__link navbar__link--active" href="/NeuroNotes/docs/Sequencing the Connectome/Papers">Notes</a></div><div class="navbar__items navbar__items--right"><div class="toggle_iYfV toggle_2i4l toggleDisabled_xj38"><div class="toggleTrack_t-f2" role="button" tabindex="-1"><div class="toggleTrackCheck_mk7D"><span class="toggleIcon_pHJ9">ğŸŒœ</span></div><div class="toggleTrackX_dm8H"><span class="toggleIcon_pHJ9">ğŸŒ</span></div><div class="toggleTrackThumb_W6To"></div></div><input type="checkbox" class="toggleScreenReader_h9qa" aria-label="Switch between dark and light mode"></div></div></div><div role="presentation" class="navbar-sidebar__backdrop"></div></nav><div class="main-wrapper docs-wrapper docs-doc-page"><div class="docPage_lDyR"><button aria-label="Scroll back to top" class="clean-btn theme-back-to-top-button backToTopButton_i9tI" type="button"></button><aside class="docSidebarContainer_0YBq"><div class="sidebar_a3j0"><nav class="menu thin-scrollbar menu_cyFh"><ul class="theme-doc-sidebar-menu menu__list"><li class="theme-doc-sidebar-item-category theme-doc-sidebar-item-category-level-1 menu__list-item"><div class="menu__list-item-collapsible"><a class="menu__link menu__link--sublist menu__link--active hasHref_TwRn" href="/NeuroNotes/docs/Sequencing the Connectome/Papers">Sequencing the Connectome</a></div><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-2 menu__list-item"><a class="menu__link" tabindex="0" href="/NeuroNotes/docs/Sequencing the Connectome/Papers">Papers</a></li><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-2 menu__list-item"><a class="menu__link" tabindex="0" href="/NeuroNotes/docs/Sequencing the Connectome/2012 Sequencing the Connectome">2012 Sequencing the Connectome</a></li><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-2 menu__list-item"><a class="menu__link" tabindex="0" href="/NeuroNotes/docs/Sequencing the Connectome/2014 In vivo generation of DNA sequence diversity for cellular barcoding">2014 In vivo generation of DNA sequence diversity for cellular barcoding</a></li><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-2 menu__list-item"><a class="menu__link" tabindex="0" href="/NeuroNotes/docs/Sequencing the Connectome/2016 High-throughput mapping of single neuron projections by sequencing of barcoded RNA">2016 High-throughput mapping of single neuron projections by sequencing of barcoded RNA</a></li><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-2 menu__list-item"><a class="menu__link menu__link--active" aria-current="page" tabindex="0" href="/NeuroNotes/docs/Sequencing the Connectome/2017 Using high-throughput barcode sequencing to efficiently map connectomes">2017 Using high-throughput barcode sequencing to efficiently map connectomes</a></li><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-2 menu__list-item"><a class="menu__link" tabindex="0" href="/NeuroNotes/docs/Sequencing the Connectome/2019 Network cloning using DNA barcodes">2019 Network cloning using DNA barcodes</a></li></ul></li></ul></nav></div></aside><main class="docMainContainer_r8cw"><div class="container padding-top--md padding-bottom--lg"><div class="row"><div class="col docItemCol_zHA2"><div class="docItemContainer_oiyr"><article><div class="tocCollapsible_aw-L theme-doc-toc-mobile tocMobile_Tx6Y"><button type="button" class="clean-btn tocCollapsibleButton_zr6a">On this page</button></div><div class="theme-doc-markdown markdown"><header><h1>2017 Using high-throughput barcode sequencing to efficiently map connectomes</h1></header><ul><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5499584/" target="_blank" rel="noopener noreferrer">Paper</a></li><li>SYNseq: </li></ul><h3 class="anchor anchorWithStickyNavbar_y2LR" id="prerequisites">prerequisites<a class="hash-link" href="#prerequisites" title="Direct link to heading">â€‹</a></h3><ul><li><p>in situ: åœ¨ç”Ÿç‰©å­¦å®éªŒä¸Šï¼Œin situæ˜¯æŒ‡è¿›è¡ŒäºåŸå‘ç”Ÿä½ç½®çš„è¯•éªŒï¼ˆè€Œä¸æ˜¯å°†å…¶ç§»å…¥ç‰¹æ®ŠåŸ¹å…»åŸºä¸­ï¼‰ï¼Œæœ‰æ—¶æ„ä¹‰å¤§è‡´ä»‹äºin vivoä¸in vitroä¹‹é—´ã€‚å½“ç ”ç©¶è€…å¯¹æŸä¸€å¤„äºäººå·¥ç¯å¢ƒä¸‹çš„å™¨å®˜ä¸­ä¹‹ç‰¹å®šç»†èƒä½œå®éªŒæ—¶ï¼Œå°±å¯ç§°ä¸ºin situã€‚ç”±äºä¸åœ¨å®Œæ•´æ´»ä½“ä¸­ï¼Œå› æ­¤ä¸æ˜¯in vivoï¼›ä¹Ÿä¸æ˜¯å•çº¯ä»¥äººå·¥ç¯å¢ƒä¸‹çš„ç»†èƒä½œå®éªŒï¼Œå› æ­¤ä¹Ÿä¸æ˜¯in vitroã€‚ä¾‹å¦‚åˆ†å­ç”Ÿç‰©å­¦ä¸Šä¸€ç§ç”¨æ¥æµ‹å®šç‰¹å®šæ ¸é…¸åºåˆ—ä½ç½®çš„åŸä½æ‚äº¤ï¼ˆIn situ hybridizationï¼‰æŠ€æœ¯ï¼Œå°±æ˜¯ä¸€ç§é’ˆå¯¹ç‰¹å®šç»„ç»‡æ‰€ä½œçš„in situå®éªŒã€‚</p></li><li><p>å…ç–«æ²‰æ·€(Immunoprecipitation, IP): æ°´æº¶æ€§æŠ—åŸé‡åˆ°ç‰¹å¼‚æ€§æŠ—ä½“ï¼Œç»“åˆè€Œå½¢æˆæŠ—åŸ-æŠ—ä½“å¤åˆç‰©åœ¨æ°´æº¶æ¶²ä¸­æ²‰æ·€çš„ç°è±¡</p></li><li><p>elution: is the process of extracting one material from another by washing with a solvent</p></li><li><p>epitope: the part of an antigen molecule to which an antibody attaches itself</p></li><li><p>mCherry: is a member of the mFruits family of monomeric <strong>red fluorescent proteins (mRFPs)</strong></p></li><li><p>HEK cells: Human embryonic kidney 293 cells, are a specific <strong>immortalised</strong> cell line.</p></li><li><p>Western blot: è›‹ç™½è´¨è½¬æ¸æ³•, åˆ©ç”¨ç‰¹å®šæŠ—ä½“èƒ½å¤Ÿä¸“ä¸€ç»“åˆå…¶æŠ—åŸè›‹ç™½è´¨çš„åŸç†æ¥å¯¹æ ·å“è¿›è¡Œç€è‰²ï¼Œé€šè¿‡åˆ†æç€è‰²çš„ä½ç½®å’Œç€è‰²æ·±åº¦è·å¾—<strong>ç‰¹å®šè›‹ç™½è´¨åœ¨æ‰€åˆ†æçš„ç»†èƒæˆ–ç»„ç»‡ä¸­çš„è¡¨è¾¾æƒ…å†µçš„ä¿¡æ¯</strong>ï¼Œæ¥åˆ†ææ£€æµ‹ç‰¹å®šè›‹ç™½è´¨çš„ç”Ÿç‰©å­¦æ£€æµ‹æŠ€æœ¯ã€‚</p></li><li><p>ligation: the joining of two DNA strands or other molecules by a phosphate ester linkage ç£·é…¸é…¯é”®.</p></li><li><p>assay: åŒ–éªŒ</p></li></ul><h3 class="anchor anchorWithStickyNavbar_y2LR" id="abstract">ABSTRACT<a class="hash-link" href="#abstract" title="Direct link to heading">â€‹</a></h3><ul><li>In SYNseq, each neuron is labeled with a unique random nucleotide sequenceâ€”an RNA &#x27;barcode&#x27;â€”which is targeted to the synapse using engineered proteins. <ul><li>Barcodes in pre- and postsynaptic neurons are then associated through protein-protein crosslinking across the synapse, extracted from the tissue, and joined into a form suitable for sequencing. </li><li>Although our <strong>failure to develop an efficient barcode joining scheme</strong> precludes the widespread application of this approach, we expect that with further development SYNseq will enable tracing of complex circuits at high speed and low cost.</li></ul></li></ul><h3 class="anchor anchorWithStickyNavbar_y2LR" id="introduction">INTRODUCTION<a class="hash-link" href="#introduction" title="Direct link to heading">â€‹</a></h3><div class="ctr"><img src="/NeuroNotes/img/p5.png" width="100%"></div><ul><li>(A) First, we express random mRNA barcodes and modified synaptic proteins in a separate population of pre- and postsynaptic neurons, so that each neuron is <strong>uniquely</strong> labeled with a different barcode sequence (compare ref (17,18)).<ul><li>17: <strong>NOT unique</strong> <code>2016 MAPseq</code> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6640135/" target="_blank" rel="noopener noreferrer">High-Throughput Mapping of Single-Neuron Projections by <strong>Sequencing</strong> of <strong>barcoded</strong> RNA.</a></li><li>18: <strong>unique</strong> <code>2016 Virus</code> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4877524/" target="_blank" rel="noopener noreferrer">A New Defective Helper RNA to Produce Recombinant Sindbis Virus that Infects Neurons but does not Propagate.</a></li></ul></li></ul><div class="ctr"><h3>2016 MAPseq</h3><img src="/NeuroNotes/img/p6.png" width="300"></div><ul><li><p>The modified synaptic proteins specifically bind the mRNA barcodes via an RNA binding domain, thereby trafficking barcodes to the pre- (left) or postsynaptic (right) compartments, respectively. The proteins meet at the synapse. </p></li><li><p>(B) Next, the pre- and postsynaptic proteins are cross-linked in situ using a custom synthetic bivalent linker, which covalently joins the extracellular domains of the pre- and postsynaptic proteins. The resulting complexes, consisting of covalently bound pre- and postsynaptic protein pairs bound via the RNA binding domain to their respective barcodes, are then purified via immunoprecipitation (IP). </p></li><li><p>(C) The associated barcode pairs, which represent pairs of connected neurons, are joined, amplified, and sequenced. </p></li><li><p>(D) Sequencing data then allows the reconstruction of a connectivity matrix.</p></li></ul><hr><ul><li><p>SYNseq consists of four steps: </p><ul><li>neuronal barcoding </li><li>trafficking of barcodes to the synapses via tight association with engineered synaptic proteins <ul><li>a pre-synaptic mRNA barcode is trafficked to the presynaptic terminal via association with an engineered version of the Neurexin1B (Nrx1B) protein </li><li>Likewise, the postsynaptic barcode is trafficked to the postsynaptic terminal via association with a modified Neuroligin1AB (Nlg1AB) protein</li></ul></li><li>joining of barcodes into a form suitable for sequencing<ul><li>Across a synapse, the presynaptic SYNseq components are covalently linked to the postsynaptic SYNseq components and then immunoprecipitated for further biochemical manipulation to link the pre- and postsynaptic barcodes.</li></ul></li><li>and reconstruction of the network connectivity </li></ul></li><li><p>We develop a set of reagents that allow for synaptic trafficking of barcodes, specific crosslinking of carrier proteins across synapses, and recovery and joining of the barcode pairs that define a specific synapse. </p></li><li><p><strong>Unfortunately, however, the efficiency of the last stepâ€”barcode joiningâ€”is in our hands insufficient to provide for reliable synapse recovery while avoiding false positives.</strong></p></li><li><p>Despite this shortcoming, our work lays the foundation for using high-throughput DNA sequencing for circuit reconstruction. Future improvements upon SYNseq have the potential to enable high-throughput connectomics, opening up many new avenues of research in neuroscience.</p></li></ul><h3 class="anchor anchorWithStickyNavbar_y2LR" id="materials-and-methods">MATERIALS AND METHODS<a class="hash-link" href="#materials-and-methods" title="Direct link to heading">â€‹</a></h3><h1>TODO</h1><h3 class="anchor anchorWithStickyNavbar_y2LR" id="results">RESULTS<a class="hash-link" href="#results" title="Direct link to heading">â€‹</a></h3><h4 class="anchor anchorWithStickyNavbar_y2LR" id="neuronal-barcodes">Neuronal barcodes<a class="hash-link" href="#neuronal-barcodes" title="Direct link to heading">â€‹</a></h4><p>We have previously developed a Sindbis-based system for over-expression of neuronal barcodes, which was used to map projections (<code>MAPseq</code>). This work revealed that unique barcodes can be efficiently targeted to individual neurons, and <strong>amplified through the Sindbis virus system</strong>. In this expression system each virus particle carries a unique RNA barcode, which is <strong>extensively replicated after infection by the machinery of the Sindbis virus</strong>, and is flanked by sequences that allow for efficient recovery and sequencing. In this work, we sought to extend the MAPseq system by developing <strong>proteins that could specifically target barcodes to the pre- and post-synaptic density, and enable recovery of barcode pairs across individual synapses.</strong></p><h4 class="anchor anchorWithStickyNavbar_y2LR" id="protein-design">Protein design<a class="hash-link" href="#protein-design" title="Direct link to heading">â€‹</a></h4><ul><li>In order to enable recovery of synaptic connectivity from neuronal barcodes, we developed a pair of SYNseq carrier proteins that conform to three requirements. <ul><li>First, we required that they traffic to the pre- and postsynaptic terminals, respectively. <ul><li>we started with the presynaptic protein <code>Nrx1B</code> and the postsynaptic protein <code>Nlg1AB</code>. Both are relatively simple, single pass membrane proteins that <strong>traffic naturally to the appropriate side of a mammalian synapse.</strong></li></ul></li><li>Second, we required that the pre- and postsynaptic components be in a form that allowed for synapse-specific covalent cross-linking. <ul><li>To permit crosslinking of the proteins across the synapse, we fused the self-labeling proteins <code>CLIP</code> and <code>SNAP</code> to the extracellular domains of <code>Nrx1B</code> and <code>Nlg1AB</code>, respectively. <code>CLIP</code> and <code>SNAP</code> are engineered versions of a <strong>DNA repair enzyme that specifically react and form a covalent bond with benzylcytosine (BC) and benzylguanine (BG), respectively.</strong> To crosslink the <code>CLIP</code> and <code>SNAP</code> domains, we synthesized a bi-functional small molecule that contains <code>BC</code> on one end, and <code>BG</code> on the other, and is <strong>thus capable of forming a covalent bond between our pre- and postsynaptic proteins.</strong> In addition, this crosslinker contains a <code>biotin</code> ç”Ÿç‰©ç´  functional group that allows efficient immunoprecipitation (IP). A second version of the crosslinker further allows for elution from beads after IP for biotin via a cleavable disulfide bridge å¯è£‚è§£äºŒç¡«é”®. The two versions of the crosslinker (<code>BGâ€“PEGâ€“biotinâ€“PEGâ€“BC</code> and <code>BGâ€“PEGâ€“(Sâ€“S)â€“biotinâ€“PEGâ€“BC</code>) behave largely identically and we used them interchangeably in this study, unless elution was necessary. Finally, we fused a <code>Myc</code> epitope tag to <code>Nrx1B</code>, and an <code>HA</code> epitope tag to <code>Nlg1AB</code>, to aid in biochemistry and/or visualization.</li></ul></li><li>Finally, we required that they bind the barcode mRNA tightly and specifically.<ul><li>To enable these proteins to carry barcode mRNAs, we inserted the RNA binding domain <code>nÎ»</code> into the <strong>cytoplasmic tail</strong> of the two proteins. The <code>nÎ»</code> domain is a 22 amino acid peptide (derived from the <code>Î»N</code> protein of <code>Î»</code> bacteriophage å™¬èŒä½“) that <strong>specifically and strongly binds to a particular 15 nt RNA sequence, termed <code>boxB</code>.</strong> We also added the corresponding <code>boxB</code> sequence to the barcode mRNA, causing the association of SYNseq proteins and barcode mRNAs.</li></ul></li></ul></li></ul><p>The design of proteins that fulfilled all of the above requirements necessitated considerable troubleshooting. We tested various positions, RNA binding domains, and linker structures before finding a pair of proteins in which our modifications to Nrx1B and Nlg1AB did not disrupt the proteins&#x27; endogenous trafficking pattern. For ease of use and fast turnaround, we performed initial troubleshooting and benchmarking of different constructs by expression and co-expression in HEK cell culture. We then tested promising proteins for the more stringent criteria of synaptic localization and RNA binding in neuronal culture.</p><h4 class="anchor anchorWithStickyNavbar_y2LR" id="barcode-mrna-design">Barcode mRNA design<a class="hash-link" href="#barcode-mrna-design" title="Direct link to heading">â€‹</a></h4><p>The barcode mRNAs necessary for SYNseq underwent few changes from the initial design. Both pre- and postsynaptic barcode RNA carry a random 30 nt barcode sequence in their <code>5&#x27; UTR</code>, and four copies of the <code>boxB</code> sequence, which allows tight association of the barcode RNA to the SYNseq proteins, in the <code>3&#x27; UTR</code>. Finally, we introduced the coding sequences for <code>GFP</code> and <code>mCherry</code> into the presynaptic and postsynaptic barcode mRNAs, respectively.</p><details class="details_Q743 alert alert--info details_h+cY" data-collapsed="true"><summary><p>Step 1: SYNseq proteins <code>traffic</code> to the membrane, can be <code>crosslinked</code> and <code>bind</code> barcode mRNA in <strong>HEK cell culture</strong></p></summary><div><div class="collapsibleContent_K5uX"><div class="ctr"><img src="/NeuroNotes/img/p14.png" width="300"></div><p>After troubleshooting, we focused on two proteins, <code>Myc-CLIP-Nrx1B-1xnÎ» (synPRE-P)</code> and <code>HA-SNAP-Nlg1AB-1xnÎ» (synPOST-P)</code>, for further testing. Each contained a single repeat of <code>nÎ»</code> surrounded by long flexible linker sequences in their cytoplasmic tail, in a position that was previously reported not to disrupt the proteins&#x27; endogenous trafficking. The matching barcode mRNAs encode for <code>GFP</code> and <code>mCherry</code>, respectively, contain a <code>random 30 nt barcode sequence</code> in their <code>5Î„ UTR</code>, and have four repeats of the <code>boxB</code> stem loop in their <code>3Î„UTR</code>.</p><ul><li><p>We first examined the localization of the two proteins. Both <code>synPRE-P</code> and <code>synPOST-P</code> traffic to the HEK cell membrane, as demonstrated by surface staining. This indicates that at least rudimentary <strong>trafficking</strong> of the proteins is uninterrupted by the addition of crosslinking and RNA binding domains.</p></li><li><p>We next tested whether we could specifically crosslink <code>synPRE-P</code> and <code>synPOST-P</code> in a simple scenario. We incubated HEK cells expressing <code>synPRE-P</code> and/or <code>synPOST-P</code> with the crosslinker, and performed an <code>IP</code> for the biotin on the crosslinker. We only detect a &#x27;crosslinked&#x27; <code>synPRE-Pâ€”synPOST-P</code> band (<strong>arrow</strong>) in addition to the signals of the individual <code>synPRE-P</code> and <code>synPOST-P</code> proteins (<strong>star</strong>), when both synPRE-P and synPOST-P proteins are expressed in the cells, and the crosslinker is added before cell lysis. Importantly, we do not detect this crosslinked band when only one of the two proteins is expressed or when the second protein was added post-lysis. Moreover, the crosslinker is inactive in the lysis buffer and does not result in nonspecific bands. These results <strong>indicate that we can form covalent synPRE-Pâ€”synPOST-P protein complexes</strong>, and that post-lysis è£‚è§£å association of synPRE-P and synPOST-P is low.</p></li></ul><div class="ctr"><img src="/NeuroNotes/img/p15.png" width="300"></div><ul><li>Finally, we confirmed that <code>synPRE-P</code> and <code>synPOST-P</code> specifically bind to barcode mRNA. After expression of either <code>synPRE-P</code> or <code>synPOST-P</code> in HEK cell culture and incubation with the biotinylated ç”Ÿç‰©ç´ åŒ– crosslinker, we find that barcode RNA is specifically enriched &gt;32-fold by a pulldown for the biotinylated crosslinker. Importantly, the protein-RNA interaction depends on the presence of both the RNA binding domain <code>nÎ»</code> and its RNA recognition sequence <code>boxB</code>.<ul><li><code>synPRE-P</code> and <code>synPOST-P</code> <strong>specifically and strongly</strong> bind to their respective barcode mRNAs as evident in RNA-IPs from transiently transfected HEK cells, after membrane tagging with <code>BGâ€“PEGâ€“biotinâ€“PEGâ€“BC</code>. We show <code>qRT-PCR</code> analysis of three independent RNA-IP experiments and western blot analysis of a representative sample.</li></ul></li></ul><p>From these experiments, we <strong>confirm that <code>synPRE-P</code> and <code>synPOST-P</code> fulfill basic requirements for SYNseq proteins in HEK cell culture: membrane trafficking, efficient crosslinking and barcode mRNA binding.</strong> To further confirm these properties, we turned to more sensitive assays using neuronal cell culture.</p></div></div></details><details class="details_Q743 alert alert--info details_h+cY" data-collapsed="true"><summary><p>Step 2: SYNseq proteins <code>traffic</code> to synapses in <strong>neuronal cell culture</strong> and can be <code>crosslinked</code> into transsynaptic complexes. (no need to check again <code>binding</code> barcode mRNA in this step)</p></summary><div><div class="collapsibleContent_K5uX"><ul><li>check trafficking<ul><li>We used a double promoter Sindbis virus to express both the synPRE-P or synPOST-P protein and the pre- or postsynaptic barcode mRNA from a single virus, ensuring that SYNseq protein expression is always coupled with barcode mRNA expression. </li><li>Sindbis virus is a positive strand RNA virus, characterized by a large (up to 6 kb) payload as well as rapid and strong expression of transgenes. It therefore allows for rapid turnaround and iteration of constructs during troubleshooting. </li><li>In addition, we have previously shown that Sindbis virus can be used to produce high diversity barcode libraries and can uniquely label neurons. The Sindbis virus was thus well-suited to act as an expression vehicle for assessing SYNseq feasibility. </li><li>Surface staining for Sindbis-expressed SYNseq proteins with CLIP-Surface488 (synPRE-P) and SNAP-Surface488 (synPOST-P) <strong>confirmed that our protein modifications did not interfere with proper trafficking</strong> of each protein to the cell membrane in neurons.</li></ul></li></ul><div class="ctr"><img src="/NeuroNotes/img/p16.png" width="100%"></div><ul><li>check crosslinking <ul><li>We then used the proximity ligation assay (PLA) to screen for protein pairs that interact across synapses. PLA is a double antibody stain that results in a highly amplified signal only if the two probed epitopes are within 40 nm of each other. This interaction length is comparable to the dimensions of a synaptic cleft é—´éš™(20 nm). The presence of a PLA signal between a pre- and a postsynaptic protein expressed in separate, but synaptically coupled populations of neurons thus provided a <strong>necessary but not sufficient criterion for establishing the synaptic localization of the two proteins</strong>. These experiments thus served as a screen for candidate protein pairs to be later tested in more stringent assays that restrict the interaction distance to &lt;5 nm.</li><li>To achieve expression of <code>synPRE-P</code> and <code>synPOST-P</code> in separate, but synaptically coupled populations of neurons, we grew cultured primary hippocampal neurons in a <code>XONA</code> microfluidic device. In a XONA, two chambers containing primary neurons are connected via long, thin microfluidic grooves å‡¹æ§½. The setup ensures fluidic isolation of the two neural populations, allowing independent viral infection of the two populations, while permitting the growth of axons between chambers through the groves.</li><li>We infected one population of cells with Sindbis virus expressing the pre-synaptic components and the other population of cells with Sindbis virus expressing the post-synaptic components. Using <strong>PLA</strong> we detected interaction between the <code>synPRE-P</code> and <code>synPOST-P</code> proteins preferentially where presynaptic (GFP positive) axons reach the postsynaptic chamber, <strong>suggesting that our proteins come within at least 40 nm of each other across separate but synaptically coupled cell populations.</strong></li><li>PLA was a rapid and convenient screening tool to troubleshoot protein design. However, its interaction distance of 40 nm is too large to confirm synaptic co-localization of synPRE-P and POST. </li><li>In contrast, the <strong>bifunctional crosslinker designed to create the trans-synaptic synPRE-Pâ€”synPOST-P complex is only 4 nm long</strong>. Thus, the creation of a synPRE-Pâ€”synPOST-P complex across synaptically coupled neuronal populations necessarily implies that the two proteins came within 4 nm of each other, satisfying even the strictest distance-based definitions of a synapse. <strong>Again expressing synPRE-P and synPOST-P in synaptically coupled but separate neuronal populations using a XONA device, we found that we could specifically crosslink and immunoprecipitate synPRE-P and synPOST-P across the two populations of neurons, and by that logic, synapses.</strong> Note that, to simplify biochemistry from neuronal culture, we used a version of synPRE-P that contains a <code>3xFLAG</code> epitope tag in addition to the <code>myc</code> epitope tag in this and all subsequent experiments.</li><li>When we immunoprecipitated crosslinked neuron samples from XONA cultures using the biotin tag on the crosslinker, we can detect at least three major products. </li><li>First, we recover the desired <code>synPRE-Pâ€”synPOST-P</code> complex formed by interaction of the proteins across the synaptic cleft. <strong>However, we also observe signals arising from single synPRE-P and synPOST-P proteins</strong> that reacted with the one side of the crosslinker, but which have no partner on the other side of the crosslinker. Such unpaired single proteins occur from at least three processes: from incomplete crosslinking across a synapse, from synapses at which only one side expresses a SYNseq protein, as well as from expression of either synPRE-P or synPOST-P in regions of the neuronal membrane that are not part of a synapse. <strong>Unfortunately, these unpaired proteins present a challenge to many barcode mRNA joining strategies.</strong> We therefore sought to remove them from the IP reaction by performing a double IP, selecting first for the postsynaptic HA tag, then for the presynaptic <code>3xFLAG</code> tag, so that only proteins that contain both synPRE-P and synPOST-P remain after the second IP step. <strong>This strategy efficiently removed all unpaired synPRE-P or synPOST-P proteins from the IP and enriches for the transsynaptic protein complex.</strong></li></ul></li><li>check binding<ul><li>no need since this is independant of the cell type, already checked by <strong>HEK cell culture</strong></li></ul></li><li>From these results, we conclude that <code>synPRE-P</code> and <code>synPOST-P</code> conform to all three requirements that we had set out for SYNseq proteins: They specifically bind to barcode mRNA, traffic to synapses and can be specifically joined into a transsynaptic protein complex.</li></ul><div class="ctr"><h3>check crosslinking using PLA (40nm)</h3><img src="/NeuroNotes/img/p17.png" width="100%"><h3>check crosslinking using bifunctional crosslinker (4nm)</h3><img src="/NeuroNotes/img/p18.png" width="300"></div></div></div></details><h4 class="anchor anchorWithStickyNavbar_y2LR" id="joining-pre--and-postsynaptic-barcode-mrnas-by-emulsion-rt-pcr">Joining pre- and postsynaptic barcode mRNAs by emulsion RT-PCR<a class="hash-link" href="#joining-pre--and-postsynaptic-barcode-mrnas-by-emulsion-rt-pcr" title="Direct link to heading">â€‹</a></h4><ul><li>With the desired functional properties of <code>synPRE-P</code> and <code>synPOST-P</code> confirmed, we set out to develop a method for joining RNA barcodes specifically and efficiently. We tested a variety of methods including<ul><li>splinted å¤¹æ¿ ligation of <code>RNaseH</code> trimmed barcode mRNAs by <code>T4</code> RNA ligase</li><li>double-headed ribozymes æ ¸é…¶ drawing on the <code>L21</code> and the <code>Carinii</code> ribozyme </li><li>barcode joining by reverse transcription where the two barcode RNAs are priming each other </li></ul></li><li>before settling on barcode joining by <strong>emulsion ä¹³èƒ¶ reverse transcription-PCR (RT-PCR) followed by overlap PCR</strong>, which had previously proven successful for joining immunoglobulin å…ç–«çƒè›‹ç™½ mRNAs. </li><li>Briefly, <ul><li>we isolate <code>synPRE-Pâ€”synPOST-P</code> complexes by <code>IP</code> and emulsify the complexes using a microfluidic droplet system at a dilution <strong>such that each droplet contains no more than one complex</strong>. </li><li>Inside each droplet, we then reverse transcribe the presynaptic and postsynaptic barcode mRNAs and subsequently join them into a single DNA strand by overlap PCR. </li><li>We then break the emulsion, isolate the barcode pairs and sequence them on an Illumina sequencing machine.</li></ul></li></ul><div class="ctr"><img src="/NeuroNotes/img/p19.png" width="100%"></div><p>The success and failure of any method to join barcode mRNAs in SYNseq is measured not simply in the generation of barcode pairs, but critically in the ability of producing barcode pairs from mRNAs in the same complex, but not from mRNAs in different complexes or even unbound mRNAs. When using emulsion RT-PCR to join barcode mRNAs, <strong>such nonspecific joining occurs when more than one synPRE-Pâ€”synPOST-P complex, and thus more than one presynaptic or postsynaptic barcode mRNA are loaded into the same droplet</strong>. The fraction of unspecific joining events therefore ultimately depends on the concentration of complexes that are loaded into droplets and decreases with input concentration, as demonstrated by loading droplets with different concentrations of a test RNA that contains the sequences of both pre- and postsynaptic barcodes, but in inverted orientations relative to the final sequencing amplicon.</p><ul><li><p>To validate the emulsion RT-PCR approach, we first tested joining using protein-RNA complexes derived from <strong>HEK cell culture</strong>. </p><ul><li>To measure the rate of nonspecific joining of barcode mRNAs, we set up two experiments in parallel. Each experiment consists of one population of HEK cells that co-express synPRE-P, synPOST-P and barcode mRNAs. In these experiments we used an earlier version of the SYNseq proteins. <strong>The barcode mRNAs in each experiment are tagged with an additional, experiment-specific sequence, a zip code (denoted here as A or B).</strong> We crosslinked and lysed æº¶è§£ each set of cells independently, and pooled the lysates è£‚è§£ç‰©. We then immunoprecipitated <code>synPRE-Pâ€”synPOST-P</code> complexes from the mixed lysate by streptavidin é“¾éœ‰äº²å’Œç´  IP, released the complexes from the beads by cleaving åŠˆå¼€ the disulfide bridge äºŒç¡«é”® on the crosslinker under mild reducing conditions and finally joined barcode mRNAs using emulsion RT-PCR. Sequencing the resulting barcode pairs revealed <strong>42</strong> presynaptic and <strong>42</strong> postsynaptic barcodes, which were joined into 45 unique barcode pairs.</li><li>Based on the experimental setup, barcode pairs should show matching zip codes (i.e. A-A pairs or B-B pairs), as those are the only biologically possible combinations. <strong>Any mixed barcode pairs (Aâ€“B or Bâ€“A) must arise from &#x27;crossover&#x27; events and therefore represent the rate of false-positives in recovery of the correct set of synaptic connections by SYNseq.</strong> In our HEK cell experiment, we find a significant crossover rate of <strong>13.3%</strong> (barcodes from one experiment joined to barcodes of another experiment). Extrapolated to include undetectable Aâ€“A and Bâ€“B crossover events, this indicates an overall <strong>false-positive rate of 26.6%</strong>. This false-positive rate, importantly, only measures false-positives arising from biochemical procedures post-lysis (i.e. proteinâ€“RNA dissociation, insufficient dilution for emulsions, etc.). It cannot quantify the false-positive barcode pairs that result from aberrant localization of the proteins or any other in vivo effects. The false-positive rate of this experiment appears to be dominated by the emulsion input concentration, <strong>suggesting that we could reduce the false positive rate by further dilution of the complexes before emulsion.</strong></li></ul></li><li><p>We next tested emulsion RT-PCR for barcode joining in samples from <strong>neuronal cultures</strong>. </p><ul><li>We infected XONA devices with barcoded <code>synPRE-P</code> and <code>synPOST-P</code> Sindbis in the left and right chamber, respectively. Again, we set up two replicate experiments in parallel using experiment specific zip codes to differentiate the barcode mRNAs. <strong>We crosslinked, lysed and mixed the lysates from the two sets of XONA devices</strong> and performed a double immunoprecipitation to isolate synaptic <code>synPRE-Pâ€”synPOST-P</code> complexes. We then performed emulsion RT-PCR on <strong>a very dilute sample of the IP products to minimize the rate of crossover events</strong>. Sequencing the resulting barcode pairs, <strong>we identified only 2 unique combinations of pre- and postsynaptic barcodes, albeit å°½ç®¡ no false positive cross-experiment barcode pairs</strong>.</li><li><h1>!!!!!!!!!!!!!?????????????</h1></li><li>While it is likely possible to increase this number of joined barcode pairs by increasing the number of droplets made, or by slightly increasing the concentration of emulsion input, we found ourselves in a regime far from the optimal necessary to achieve high throughput connectivity mapping by SYNseq. Taken together, our emulsion results suggest, that <strong>while SYNseq can be used to read out synaptic connectivity, barcode joining is currently a very low efficiency process, which will need to be optimized further before SYNseq can be used for circuit mapping.</strong></li></ul></li></ul><h3 class="anchor anchorWithStickyNavbar_y2LR" id="discussion">DISCUSSION<a class="hash-link" href="#discussion" title="Direct link to heading">â€‹</a></h3><h4 class="anchor anchorWithStickyNavbar_y2LR" id="dna-sequencing-for-neuroanatomy">DNA sequencing for neuroanatomy<a class="hash-link" href="#dna-sequencing-for-neuroanatomy" title="Direct link to heading">â€‹</a></h4><p>SYNseq takes inspiration from GRASP/mGRASP, chromosome confirmation capture and synaptic brainbow. </p><h4 class="anchor anchorWithStickyNavbar_y2LR" id="challenges-ahead">Challenges ahead<a class="hash-link" href="#challenges-ahead" title="Direct link to heading">â€‹</a></h4><ul><li>We concluded by a process of elimination that <strong>it is barcode joining</strong> that is limiting our ability to detect synaptic connections by SYNseq. </li><li>Barcoding of neurons and HEK cells is a high efficiency process. </li><li>We further demonstrated that barcode RNA and SYNseq proteins interact strongly and specifically, and that we can produce a significant number of trans-synaptic <code>synPRE-Pâ€”synPOST-P</code> complexes. </li><li>Despite this, we are unable to generate a comparable number of joined barcode pairs, which we therefore blame on inefficiencies of the droplet based joining strategy.</li><li>We have not attempted to directly de-convolve the inefficiencies of reverse transcription vs. overlap PCR, but preliminary evidence place our in-droplet RT efficiency on the order of 1%. Such low RT efficiency will have a dramatic impact on our ability to produce joint barcode pairs, as both pre and postsynaptic barcode mRNAs need to be successfully reverse transcribed in each successfully loaded droplet (this will happen in only 0.01^2 of all properly loaded droplets). Given these considerations, we predict reverse transcription, not overlap PCR, which has a chance to join the two barcodes at every cycle of PCR, to be limiting the reaction. Although we were unable to achieve efficient and specific joining of barcodes in droplets, it is possible that with further optimization a droplet-based strategy, perhaps in combination with barcoded beads, might succeed. </li><li>Alternatively, joining strategies based on RNA ligation, gap filling or immobilization of complexes on beads, chips or in gels might emerge as more efficient strategies. </li><li>Finally, sequencing assays relying on the joining of barcodes (albeit in DNA form) are being developed for other applications, suggesting that in the future new technologies for barcode joining may emerge. Thus, although our present inability to achieve efficient and specific barcode joining limits the practical application of this approach, we expect the approach we have presented will form that foundation for an efficient method for deducing synaptic connections.</li></ul><h4 class="anchor anchorWithStickyNavbar_y2LR" id="space-and-neuroanatomy">Space and neuroanatomy<a class="hash-link" href="#space-and-neuroanatomy" title="Direct link to heading">â€‹</a></h4><h4 class="anchor anchorWithStickyNavbar_y2LR" id="future-directions">Future directions<a class="hash-link" href="#future-directions" title="Direct link to heading">â€‹</a></h4><p>We envision that SYNseq will serve as a platform for future development toward encoding neural connectivity, and/or other biological variablesâ€”including cellular activity and/or identityâ€”into a form that can be read by high-throughput sequencing, a technology which already operates at the scale of the complexity of neural circuits. </p><p>Improvements to this method that allow for the co-expression of all of the components within the same cellsâ€”for example, by employing two separate RNA binding domains â€”will allow for bi-directional and local-circuit mapping. </p><p>Transgenic techniques, combined with an in vivo barcoding scheme based on recombinases or CRISPR cas9 will allow for scaling this technology to mapping full brains.</p></div><footer class="theme-doc-footer docusaurus-mt-lg"><div class="theme-doc-footer-edit-meta-row row"><div class="col"><a href="https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/docs/Sequencing the Connectome/2017 Using high-throughput barcode sequencing to efficiently map connectomes.md" target="_blank" rel="noreferrer noopener" class="theme-edit-this-page"><svg fill="currentColor" height="20" width="20" viewBox="0 0 40 40" class="iconEdit_mS5F" aria-hidden="true"><g><path d="m34.5 11.7l-3 3.1-6.3-6.3 3.1-3q0.5-0.5 1.2-0.5t1.1 0.5l3.9 3.9q0.5 0.4 0.5 1.1t-0.5 1.2z m-29.5 17.1l18.4-18.5 6.3 6.3-18.4 18.4h-6.3v-6.2z"></path></g></svg>Edit this page</a></div><div class="col lastUpdated_mt2f"></div></div></footer></article><nav class="pagination-nav docusaurus-mt-lg" aria-label="Docs pages navigation"><div class="pagination-nav__item"><a class="pagination-nav__link" href="/NeuroNotes/docs/Sequencing the Connectome/2016 High-throughput mapping of single neuron projections by sequencing of barcoded RNA"><div class="pagination-nav__sublabel">Previous</div><div class="pagination-nav__label">2016 High-throughput mapping of single neuron projections by sequencing of barcoded RNA</div></a></div><div class="pagination-nav__item pagination-nav__item--next"><a class="pagination-nav__link" href="/NeuroNotes/docs/Sequencing the Connectome/2019 Network cloning using DNA barcodes"><div class="pagination-nav__sublabel">Next</div><div class="pagination-nav__label">2019 Network cloning using DNA barcodes</div></a></div></nav></div></div><div class="col col--3"><div class="tableOfContents_vrFS thin-scrollbar theme-doc-toc-desktop"><ul class="table-of-contents table-of-contents__left-border"><li><a href="#prerequisites" class="table-of-contents__link toc-highlight">prerequisites</a></li><li><a href="#abstract" class="table-of-contents__link toc-highlight">ABSTRACT</a></li><li><a href="#introduction" class="table-of-contents__link toc-highlight">INTRODUCTION</a></li><li><a href="#materials-and-methods" class="table-of-contents__link toc-highlight">MATERIALS AND METHODS</a></li><li><a href="#results" class="table-of-contents__link toc-highlight">RESULTS</a></li><li><a href="#discussion" class="table-of-contents__link toc-highlight">DISCUSSION</a></li></ul></div></div></div></div></main></div></div><footer class="footer footer--dark"><div class="container container-fluid"><div class="footer__bottom text--center"><div class="footer__copyright">Copyright Â© 2022 NeuroNotes, Inc. Built with Docusaurus.</div></div></div></footer></div>
<script src="/NeuroNotes/assets/js/runtime~main.1c73bbed.js"></script>
<script src="/NeuroNotes/assets/js/main.31e959ff.js"></script>
</body>
</html>