<!doctype html>
<html class="docs-version-current" lang="en" dir="ltr">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width,initial-scale=1">
<meta name="generator" content="Docusaurus v0.0.0-4406">
<link rel="alternate" type="application/rss+xml" href="/NeuroNotes/blog/rss.xml" title="NeuroNotes RSS Feed">
<link rel="alternate" type="application/atom+xml" href="/NeuroNotes/blog/atom.xml" title="NeuroNotes Atom Feed"><title data-react-helmet="true">Papers | NeuroNotes</title><meta data-react-helmet="true" name="twitter:card" content="summary_large_image"><meta data-react-helmet="true" property="og:url" content="https://tesla-cat.github.io/NeuroNotes/docs/Sequencing the Connectome/Papers"><meta data-react-helmet="true" name="docusaurus_locale" content="en"><meta data-react-helmet="true" name="docusaurus_version" content="current"><meta data-react-helmet="true" name="docusaurus_tag" content="docs-default-current"><meta data-react-helmet="true" property="og:title" content="Papers | NeuroNotes"><meta data-react-helmet="true" name="description" content="Sequencing the connectome"><meta data-react-helmet="true" property="og:description" content="Sequencing the connectome"><link data-react-helmet="true" rel="icon" href="/NeuroNotes/img/favicon.ico"><link data-react-helmet="true" rel="canonical" href="https://tesla-cat.github.io/NeuroNotes/docs/Sequencing the Connectome/Papers"><link data-react-helmet="true" rel="alternate" href="https://tesla-cat.github.io/NeuroNotes/docs/Sequencing the Connectome/Papers" hreflang="en"><link data-react-helmet="true" rel="alternate" href="https://tesla-cat.github.io/NeuroNotes/docs/Sequencing the Connectome/Papers" hreflang="x-default"><link rel="stylesheet" href="/NeuroNotes/assets/css/styles.9fbd5245.css">
<link rel="preload" href="/NeuroNotes/assets/js/runtime~main.9fa0cc15.js" as="script">
<link rel="preload" href="/NeuroNotes/assets/js/main.288aa2cb.js" as="script">
</head>
<body>
<script>!function(){function t(t){document.documentElement.setAttribute("data-theme",t)}var e=function(){var t=null;try{t=localStorage.getItem("theme")}catch(t){}return t}();t(null!==e?e:"light")}()</script><div id="__docusaurus">
<div role="region"><a href="#" class="skipToContent_OuoZ">Skip to main content</a></div><nav class="navbar navbar--fixed-top"><div class="navbar__inner"><div class="navbar__items"><button aria-label="Navigation bar toggle" class="navbar__toggle clean-btn" type="button" tabindex="0"><svg width="30" height="30" viewBox="0 0 30 30" aria-hidden="true"><path stroke="currentColor" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2" d="M4 7h22M4 15h22M4 23h22"></path></svg></button><a class="navbar__brand" href="/NeuroNotes/"><div class="navbar__logo"><img src="/NeuroNotes/img/logo.svg" alt="NeuroNotes Logo" class="themedImage_TMUO themedImage--light_4Vu1"><img src="/NeuroNotes/img/logo.svg" alt="NeuroNotes Logo" class="themedImage_TMUO themedImage--dark_uzRr"></div><b class="navbar__title">NeuroNotes</b></a><a aria-current="page" class="navbar__item navbar__link navbar__link--active" href="/NeuroNotes/docs/Sequencing the Connectome/Papers">Notes</a></div><div class="navbar__items navbar__items--right"><div class="toggle_iYfV toggle_2i4l toggleDisabled_xj38"><div class="toggleTrack_t-f2" role="button" tabindex="-1"><div class="toggleTrackCheck_mk7D"><span class="toggleIcon_pHJ9">ðŸŒœ</span></div><div class="toggleTrackX_dm8H"><span class="toggleIcon_pHJ9">ðŸŒž</span></div><div class="toggleTrackThumb_W6To"></div></div><input type="checkbox" class="toggleScreenReader_h9qa" aria-label="Switch between dark and light mode"></div></div></div><div role="presentation" class="navbar-sidebar__backdrop"></div></nav><div class="main-wrapper docs-wrapper docs-doc-page"><div class="docPage_lDyR"><button aria-label="Scroll back to top" class="clean-btn theme-back-to-top-button backToTopButton_i9tI" type="button"></button><aside class="docSidebarContainer_0YBq"><div class="sidebar_a3j0"><nav class="menu thin-scrollbar menu_cyFh"><ul class="theme-doc-sidebar-menu menu__list"><li class="theme-doc-sidebar-item-category theme-doc-sidebar-item-category-level-1 menu__list-item"><div class="menu__list-item-collapsible"><a class="menu__link menu__link--sublist menu__link--active hasHref_TwRn" href="/NeuroNotes/docs/Sequencing the Connectome/Papers">Sequencing the Connectome</a></div><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-2 menu__list-item"><a class="menu__link menu__link--active" aria-current="page" tabindex="0" href="/NeuroNotes/docs/Sequencing the Connectome/Papers">Papers</a></li><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-2 menu__list-item"><a class="menu__link" tabindex="0" href="/NeuroNotes/docs/Sequencing the Connectome/2012 Sequencing the Connectome">2012 Sequencing the Connectome</a></li><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-2 menu__list-item"><a class="menu__link" tabindex="0" href="/NeuroNotes/docs/Sequencing the Connectome/2014 In vivo generation of DNA sequence diversity for cellular barcoding">2014 In vivo generation of DNA sequence diversity for cellular barcoding</a></li><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-2 menu__list-item"><a class="menu__link" tabindex="0" href="/NeuroNotes/docs/Sequencing the Connectome/2016 High-throughput mapping of single neuron projections by sequencing of barcoded RNA">2016 High-throughput mapping of single neuron projections by sequencing of barcoded RNA</a></li><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-2 menu__list-item"><a class="menu__link" tabindex="0" href="/NeuroNotes/docs/Sequencing the Connectome/2017 Using high-throughput barcode sequencing to efficiently map connectomes">2017 Using high-throughput barcode sequencing to efficiently map connectomes</a></li><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-2 menu__list-item"><a class="menu__link" tabindex="0" href="/NeuroNotes/docs/Sequencing the Connectome/2019 Network cloning using DNA barcodes">2019 Network cloning using DNA barcodes</a></li></ul></li><li class="theme-doc-sidebar-item-category theme-doc-sidebar-item-category-level-1 menu__list-item menu__list-item--collapsed"><div class="menu__list-item-collapsible"><a class="menu__link menu__link--sublist hasHref_TwRn" href="/NeuroNotes/docs/fMRI/Papers">fMRI</a></div></li><li class="theme-doc-sidebar-item-category theme-doc-sidebar-item-category-level-1 menu__list-item menu__list-item--collapsed"><div class="menu__list-item-collapsible"><a class="menu__link menu__link--sublist hasHref_TwRn" href="/NeuroNotes/docs/Neuromorphic/2021 Neuromorphic electronics based on copying and pasting the brain">Neuromorphic</a></div></li><li class="theme-doc-sidebar-item-link theme-doc-sidebar-item-link-level-1 menu__list-item"><a class="menu__link" href="/NeuroNotes/docs/Links">Links</a></li></ul></nav></div></aside><main class="docMainContainer_r8cw"><div class="container padding-top--md padding-bottom--lg"><div class="row"><div class="col docItemCol_zHA2"><div class="docItemContainer_oiyr"><article><div class="tocCollapsible_aw-L theme-doc-toc-mobile tocMobile_Tx6Y"><button type="button" class="clean-btn tocCollapsibleButton_zr6a">On this page</button></div><div class="theme-doc-markdown markdown"><header><h1>Papers</h1></header><h2 class="anchor anchorWithStickyNavbar_y2LR" id="sequencing-the-connectome"><strong>Sequencing</strong> the connectome<a class="hash-link" href="#sequencing-the-connectome" title="Direct link to heading">â€‹</a></h2><div class="ctr"><img src="/NeuroNotes/img/p11.png" width="500"></div><details class="details_Q743 alert alert--info details_h+cY" data-collapsed="true"><summary><h3 class="anchor anchorWithStickyNavbar_y2LR" id="neuron-neuron-synapse-connectome">Neuron-neuron synapse (connectome)<a class="hash-link" href="#neuron-neuron-synapse-connectome" title="Direct link to heading">â€‹</a></h3><ul><li><code>2012 BOINC</code>: barcoding of individual neuronal connections</li><li><code>2017 SYNSeq</code></li></ul></summary><div><div class="collapsibleContent_K5uX"><ul><li><p><code>BOINC</code> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3479097/" target="_blank" rel="noopener noreferrer"><strong>Sequencing</strong> the connectome.</a></p><ul><li>Zador AM, Dubnau J, Oyibo HK, Zhan H, Cao G, Peikon ID.</li><li>PLoS Biol. </li><li>2012</li></ul></li><li><p><code>SYNseq</code> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5499584/" target="_blank" rel="noopener noreferrer">Using high-throughput <strong>barcode</strong> <strong>sequencing</strong> to efficiently map connectomes.</a></p><ul><li>Peikon ID, Kebschull JM, Vagin VV, Ravens DI, Sun YC, Brouzes E, CorrÃªa IR Jr, Bressan D, Zador AM.</li><li>Nucleic Acids Res. </li><li>2017 Jul 7</li></ul></li><li><p><code>SYNSeq</code> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6511037/" target="_blank" rel="noopener noreferrer">Network cloning using DNA <strong>barcodes</strong>.</a></p><ul><li>Shuvaev SA, BaÅŸerdem B, Zador AM, Koulakov AA.</li><li>Proc Natl Acad Sci U S A. </li><li>2019 May 7</li></ul></li></ul></div></div></details><details class="details_Q743 alert alert--info details_h+cY" data-collapsed="true"><summary><h3 class="anchor anchorWithStickyNavbar_y2LR" id="single-neuron-projection-projectome">Single neuron projection (projectome)<a class="hash-link" href="#single-neuron-projection-projectome" title="Direct link to heading">â€‹</a></h3><ul><li><code>2016 MAPseq</code>: Multiplexed Analysis of Projections by Sequencing<ul><li>In MAPseq, neurons are barcoded using a <strong>Sindbis virus</strong> library. Both the source area containing neuronal somata and target projection areas are micro-dissected into &quot;cubelets&quot; and sequenced. Barcodes from the target areas are then matched to those at the source area to reveal projection patterns. The <strong>spatial resolution</strong> of MAPseq is thus determined by the size of the cubelets.</li></ul></li><li><code>2018 BaristaSeq</code>: Barcode <strong>in situ</strong> targeted sequencing<ul><li>an improved version of the gap padlock probe-based method with a five-fold increase in efficiency<ul><li>To capture short diverse barcode sequences, we needed an approach in which the actual target barcode sequence is captured and amplified. Although some in situ hybridization methods use sequencing to read out gene-specific tags to allow multiplexed detection of RNAs, they <strong>cannot directly sequence the RNAs of interest</strong> and are unsuitable for barcode sequencing. Circligase-based FISSEQ can readout mRNA sequences, but is not sufficiently sensitive for barcode sequencing. We therefore used a targeted approach, BaristaSeq, in which reverse transcription was used to <strong>convert the barcode sequence into cDNA</strong>, after which padlock probe hybridization followed by gap-filling and ligation was used to form a circular template for rolling circle amplification. The gap-filling padlock approach allowed for <strong>efficient amplification of barcodes</strong> in neurons co-expressing barcodes and GFP. We also adapted Illumina Sequencing By Synthesis (SBS) chemistry for in situ reactions. In combination, these strategies formed the basis for a protocol that allowed efficient and robust sequencing of barcoded <strong>cultured neurons</strong>.</li></ul></li><li><code>MAPseq</code> and others: the locations of the neurons under investigation had to be determined by manually picking individual neuronsâ€”a method which scales poorly. </li><li>Efficient barcode sequencing <strong>in situ</strong> would allow us to preserve the location of each neuron&#x27;s soma while still maintaining high-throughput lineage tracing and projection mapping. </li><li>Furthermore, in the case of <code>MAPseq</code> where the barcodes fill the neurons, in situ barcode sequencing would potentially allow visualization of the morphology of individual neurons. </li><li>Preserving the locations of the cells being sequenced would allow correlation of lineages and projections with other information, such as gene expression assayed through FISH and neuronal activities assayed through functional imaging, at cellular resolution.</li></ul></li><li><code>2019 BARseq = MAPseq + BaristaSeq</code>: Barcoded Anatomy Resolved by Sequencing<ul><li>a method that combines <code>MAPseq</code> with <strong>in situ</strong> sequencing of cellular barcodes (<code>BaristaSeq</code>).</li><li><code>MAPseq</code> protocol relies on tissue homogenization, so the precise location of each soma is lost.</li><li>In BARseq, we perform in situ sequencing of barcoded somata at the <strong>source</strong>; the target projection areas are still micro-dissected and sequenced as cubelets. BARseq thus inherits the throughput and cubelet resolution of projections of <code>MAPseq</code>, but allows the precise somatic <strong>origin</strong> of each axonal projection to be determined with <strong>cellular resolution</strong>.</li></ul></li></ul><div class="ctr"><h3>BARseq = MAPseq + BaristaSeq</h3><img src="/NeuroNotes/img/p10.png" width="500"></div><ul><li><code>2020 BRICseq = MAPseq + multi source</code>: BRain-wide Individual-animal Connectome sequencing<ul><li><code>MAPseq</code> was originally developed to study projections from a <strong>single source</strong>. Conceptually, a straightforward generalization of MAPseq to determine the projections from <strong>many source areas in the same experiment</strong> would be to tag neurons with an additional area-specific barcode sequenceâ€”a &quot;zipcode&quot;â€”which could be used to identify the source (somatic origin) of each projection. </li><li>We pursued a more convenient strategy, which requires only a single batch of virus. We hypothesized that we could reliably determine the source of each projection using only sequencing, by exploiting the higher abundance of RNA barcodes in the soma-proximal compartments (including soma and proximal dendrites) compared with the axon terminals. According to this &#x27;soma-max&#x27; strategy, the cubelet with the highest abundance of a given barcode of interest is assumed to be the source of the projection. Compared to the ground truth determined by the zipcode, the soma-max strategy correctly identified the soma location for 99.2Â±0.2% of all cells. These results indicate that the soma-max strategy would allow accurate reconstruction of connectivity even when only a <strong>single viral library</strong> is injected.</li></ul></li></ul><div class="ctr"><h3>BRICseq = MAPseq + multi source</h3><img src="/NeuroNotes/img/p9.png" width="100%"></div><ul><li><code>2021 BARseq2 = BARseq + Spatial transcriptomics</code>: Barcoded Anatomy Resolved by Sequencing 2<ul><li>In <code>BARseq</code>, each neuron is labeled with a unique virally-encoded RNA barcode that is replicated in the somas and transported to the axon terminals. </li><li>The barcodes at the axon terminals located at various target areas are sequenced and matched to somatic barcodes, which are sequenced in situ, in order to determine the projection patterns of each labeled neuron. </li><li>Because <code>BARseq</code> preserves the location of somata with high spatial resolution, in principle it provides a platform to combine projection mapping with other neuronal properties also interrogated in situ, including gene expression. We have previously shown that <code>BARseq</code> can be combined with fluorescent in situ hybridization (FISH) and Cre-labeling to uncover <strong>projections across neuronal subtypes defined by gene expression</strong>. However, these approaches <strong>can only interrogate one or a few genes at a time</strong>, which would be insufficient for unraveling the complex relationship between the expression of many genes to diverse cortical projections.</li><li>Here we aim to develop a technique to simultaneously map projections to multiple brain areas and <strong>detect the expression of dozens of genes</strong> in hundreds to thousands of neurons from a cortical area with high throughput, high spatial resolution, and cellular resolution. </li><li>To achieve this goal, we combine the high throughput and multiplexed projection mapping capability of <code>BARseq</code> with state-of-the-art <strong>spatial transcriptomic techniques with high imaging throughput and multiplexing capacity</strong>. This second-generation <code>BARseq</code> (BARseq2) greatly improves the ability to correlate the expression of many genes to projections to many targets in the same neurons. </li><li>As a proof-of-principle, we first demonstrate multiplexed gene detection using BARseq2 by mapping the spatial pattern of up to 65 cadherins and cell-type markers in 29,933 cells. We then correlate the expression of 20 cadherins to projections to up to 35 target areas in 1,349 neurons in mouse motor and auditory cortex. Our study reveals novel sets of cadherins that correlate with homologous projections in both cortical areas. BARseq2 thus bridges transcriptomic signatures obtained through spatial transcriptional profiling with sequencing-based projection mapping to illuminate the molecular logic of long-range projections.</li></ul></li></ul><div class="ctr"><h3>BARseq2 = BARseq + Spatial transcriptomics</h3><img src="/NeuroNotes/img/p12.png" width="100%"></div></summary><div><div class="collapsibleContent_K5uX"><ul><li><p><code>MAPseq</code> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6640135/" target="_blank" rel="noopener noreferrer">High-Throughput Mapping of Single-Neuron Projections by <strong>Sequencing</strong> of <strong>barcoded</strong> RNA.</a></p><ul><li>Kebschull JM, Garcia da Silva P, Reid AP, Peikon ID, Albeanu DF, Zador AM.</li><li>Neuron. </li><li>2016 Sep 7</li></ul></li><li><p><code>BaristaSeq</code> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5829746/" target="_blank" rel="noopener noreferrer">Efficient in situ <strong>barcode</strong> <strong>sequencing</strong> using padlock probe-based <strong>BaristaSeq</strong>.</a></p><ul><li>Chen X, Sun YC, Church GM, Lee JH, Zador AM.</li><li>Nucleic Acids Res. </li><li>2018 Feb 28</li></ul></li><li><p><code>MAPseq</code> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6585423/" target="_blank" rel="noopener noreferrer">The logic of single-cell projections from visual cortex.</a></p><ul><li>Han Y, Kebschull JM, Campbell RAA, Cowan D, Imhof F, Zador AM, Mrsic-Flogel TD.</li><li>Nature. </li><li>2018 Apr 5 </li></ul></li><li><p><code>MAPseq BaristaSeq</code> <a href="http://zadorlab.labsites.cshl.edu/wp-content/uploads/sites/59/2019/08/Kebschull-Zador-2018-Cellular-barcoding-lineage-tracing.pdf" target="_blank" rel="noopener noreferrer">Cellular <strong>barcoding</strong>: lineage tracing, screening and beyond.</a></p><ul><li>Kebschull JM, Zador AM.</li><li>Nat Methods. </li><li>2018 Nov</li></ul></li><li><p><code>BARseq</code> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7836778/" target="_blank" rel="noopener noreferrer">High-Throughput Mapping of Long-Range Neuronal Projection Using In Situ <strong>Sequencing</strong>.</a></p><ul><li>Chen X, Sun YC, Zhan H, Kebschull JM, Fischer S, Matho K, Huang ZJ, Gillis J, Zador AM.</li><li>Cell. </li><li>2019 Oct 17</li></ul></li><li><p><code>BRICseq</code> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7771207/" target="_blank" rel="noopener noreferrer"><strong>BRICseq</strong> Bridges Brain-wide Interregional Connectivity to Neural Activity and Gene Expression in Single Animals.</a></p><ul><li>Huang L, Kebschull JM, FÃ¼rth D, Musall S, Kaufman MT, Churchland AK, Zador AM.</li><li>Cell. </li><li>2020 Jul 9</li></ul></li><li><p><code>BARseq</code> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7971881/" target="_blank" rel="noopener noreferrer"><strong>BARcode</strong> DEmixing through Non-negative Spatial Regression (BarDensr).</a></p><ul><li>Chen S, Loper J, Chen X, Vaughan A, Zador AM, Paninski L.</li><li>PLoS Comput Biol. </li><li>2021 Mar 8</li></ul></li><li><p><code>BARseq2</code> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8178227/" target="_blank" rel="noopener noreferrer">Integrating <strong>barcoded</strong> neuroanatomy with spatial transcriptional profiling enables identification of gene correlates of projections.</a></p><ul><li>Sun YC, Chen X, Fischer S, Lu S, Zhan H, Gillis J, Zador AM.</li><li>Nat Neurosci. </li><li>2021 Jun</li></ul></li><li><p><code>BARseq</code> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8494646/" target="_blank" rel="noopener noreferrer">Cellular anatomy of the mouse primary motor cortex.</a></p><ul><li>MuÃ±oz-CastaÃ±eda R, Zingg B, Matho KS, Chen X, Wang Q, Foster NN, Li A, Narasimhan A, Hirokawa KE, Huo B, Bannerjee S, Korobkova L, Park CS, Park YG, Bienkowski MS, Chon U, Wheeler DW, Li X, Wang Y, Naeemi M, Xie P, Liu L, Kelly K, An X, Attili SM, Bowman I, Bludova A, Cetin A, Ding L, Drewes R, D&#x27;Orazi F, Elowsky C, Fischer S, Galbavy W, Gao L, Gillis J, Groblewski PA, Gou L, Hahn JD, Hatfield JT, Hintiryan H, Huang JJ, Kondo H, Kuang X, Lesnar P, Li X, Li Y, Lin M, Lo D, Mizrachi J, Mok S, Nicovich PR, Palaniswamy R, Palmer J, Qi X, Shen E, Sun YC, Tao HW, Wakemen W, Wang Y, Yao S, Yuan J, Zhan H, Zhu M, Ng L, Zhang LI, Lim BK, Hawrylycz M, Gong H, Gee JC, Kim Y, Chung K, Yang XW, Peng H, Luo Q, Mitra PP, Zador AM, Zeng H, Ascoli GA, Josh Huang Z, Osten P, Harris JA, Dong HW.</li><li>Nature. </li><li>2021 Oct</li></ul></li></ul></div></div></details><details class="details_Q743 alert alert--info details_h+cY" data-collapsed="true"><summary><h3 class="anchor anchorWithStickyNavbar_y2LR" id="toolbox">Toolbox<a class="hash-link" href="#toolbox" title="Direct link to heading">â€‹</a></h3></summary><div><div class="collapsibleContent_K5uX"><ul><li><p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4176322/" target="_blank" rel="noopener noreferrer">In vivo generation of DNA sequence diversity for cellular <strong>barcoding</strong>.</a></p><ul><li>Peikon ID, Gizatullina DI, Zador AM.</li><li>Nucleic Acids Res. </li><li>2014</li></ul></li><li><p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4153299/" target="_blank" rel="noopener noreferrer">Long-term Cre-mediated retrograde <strong>tagging</strong> of neurons using a novel recombinant pseudorabies virus.</a></p><ul><li>Oyibo HK, Znamenskiy P, Oviedo HV, Enquist LW, Zador AM.</li><li>Front Neuroanat. </li><li>2014 Sep 3</li></ul></li><li><p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4666380/" target="_blank" rel="noopener noreferrer">Sources of PCR-induced distortions in high-throughput <strong>sequencing</strong> data sets.</a></p><ul><li>Kebschull JM, Zador AM.</li><li>Nucleic Acids Res. </li><li>2015 Dec 2</li></ul></li><li><p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4877524/" target="_blank" rel="noopener noreferrer">A New Defective Helper RNA to Produce Recombinant Sindbis Virus that Infects Neurons but does not Propagate.</a></p><ul><li>Kebschull JM, Garcia da Silva P, Zador AM.</li><li>Front Neuroanat. </li><li>2016 May 24</li></ul></li></ul></div></div></details><details class="details_Q743 alert alert--info details_h+cY" data-collapsed="true"><summary>other papers from Prof. Tony Zador</summary><div><div class="collapsibleContent_K5uX"><h2 class="anchor anchorWithStickyNavbar_y2LR" id="learning">learning<a class="hash-link" href="#learning" title="Direct link to heading">â€‹</a></h2><ul><li>Corticostriatal Plasticity Established by Initial Learning Persists after Behavioral Reversal.<ul><li>Ghosh S, Zador AM.</li><li>eNeuro. </li><li>2021 Mar 11</li></ul></li><li>SYNPLA, a method to identify synapses displaying plasticity after learning.<ul><li>Dore K, Pao Y, Soria Lopez J, Aronson S, Zhan H, Ghosh S, Merrill S, Zador AM, Malinow R, Kebschull JM.</li><li>Proc Natl Acad Sci U S A. </li><li>2020 Feb 11</li></ul></li><li>A critique of pure learning and what artificial neural networks can learn from animal brains.<ul><li>Zador AM.</li><li>Nat Commun. </li><li>2019 Aug 21</li></ul></li></ul><h2 class="anchor anchorWithStickyNavbar_y2LR" id="auditory"><strong>auditory</strong><a class="hash-link" href="#auditory" title="Direct link to heading">â€‹</a></h2><ul><li><p>Selective corticostriatal plasticity during acquisition of an <strong>auditory</strong> discrimination task.</p><ul><li>Xiong Q, Znamenskiy P, Zador AM.</li><li>Nature. </li><li>2015 May 21</li></ul></li><li><p><strong>auditory</strong> thalamus and <strong>auditory</strong> cortex are equally modulated by context during flexible categorization of <strong>sounds</strong>.</p><ul><li>Jaramillo S, Borges K, Zador AM.</li><li>J Neurosci. </li><li>2014 Apr 9</li></ul></li><li><p>Corticostriatal neurons in <strong>auditory</strong> cortex drive <strong>decision</strong>s during <strong>auditory</strong> discrimination.</p><ul><li>Znamenskiy P, Zador AM.</li><li>Nature. </li><li>2013 May 23</li></ul></li><li><p>Up states are rare in awake <strong>auditory</strong> cortex.</p><ul><li>HromÃ¡dka T, Zador AM, DeWeese MR.</li><li>J Neurophysiol. </li><li>2013 Apr</li></ul></li><li><p>PTEN regulation of local and long-range connections in mouse <strong>auditory</strong> cortex.</p><ul><li>Xiong Q, Oviedo HV, Trotman LC, Zador AM.
J Neurosci. </li><li>2012 Feb 1</li></ul></li><li><p>The <strong>auditory</strong> cortex mediates the perceptual effects of acoustic temporal expectation.</p><ul><li>Jaramillo S, Zador AM.</li><li>Nat Neurosci. </li><li>2011 Feb</li></ul></li><li><p>The functional asymmetry of <strong>auditory</strong> cortex is reflected in the organization of local cortical circuits.</p><ul><li>Oviedo HV, Bureau I, Svoboda K, Zador AM.</li><li>Nat Neurosci. </li><li>2010 Nov</li></ul></li><li><p>Long-lasting context dependence constrains neural encoding models in rodent <strong>auditory</strong> cortex.</p><ul><li>Asari H, Zador AM.</li><li>J Neurophysiol. </li><li>2009 Nov</li></ul></li><li><p>Representations in <strong>auditory</strong> cortex.</p><ul><li>HromÃ¡dka T, Zador AM.</li><li>Curr Opin Neurobiol.</li><li>2009 Aug</li></ul></li><li><p>Engaging in an <strong>auditory</strong> task suppresses responses in <strong>auditory</strong> cortex.</p><ul><li>Otazu GH, Tai LH, Yang Y, Zador AM.</li><li>Nat Neurosci. </li><li>2009 May</li></ul></li><li><p>Millisecond-scale differences in neural activity in <strong>auditory</strong> cortex can drive <strong>decision</strong>s.</p><ul><li>Yang Y, DeWeese MR, Otazu GH, Zador AM.</li><li>Nat Neurosci. </li><li>2008 Nov</li></ul></li><li><p>Sparse representation of <strong>sounds</strong> in the unanesthetized <strong>auditory</strong> cortex.</p><ul><li>HromÃ¡dka T, Deweese MR, Zador AM.</li><li>PLoS Biol. </li><li>2008 Jan</li></ul></li><li><p>Toward the mechanisms of <strong>auditory</strong> attention.</p><ul><li>HromÃ¡dka T, Zador AM.</li><li>Hear Res. </li><li>2007 Jul</li></ul></li><li><p>Non-Gaussian membrane potential dynamics imply sparse, synchronous activity in <strong>auditory</strong> cortex.</p><ul><li>DeWeese MR, Zador AM.</li><li>J Neurosci. </li><li>2006 Nov 22</li></ul></li><li><p>Reliability and representational bandwidth in the <strong>auditory</strong> cortex.</p><ul><li>DeWeese MR, HromÃ¡dka T, Zador AM.</li><li>Neuron. </li><li>2005 Nov 3</li></ul></li><li><p>Neural gallops across <strong>auditory</strong> streams.</p><ul><li>Deweese MR, Zador AM.</li><li>Neuron. </li><li>2005 Oct 6</li></ul></li><li><p>Synaptic mechanisms of forward suppression in rat <strong>auditory</strong> cortex.</p><ul><li>Wehr M, Zador AM.</li><li>Neuron. </li><li>2005</li></ul></li><li><p>Shared and private variability in the <strong>auditory</strong> cortex.</p><ul><li>Deweese MR, Zador AM.</li><li>J Neurophysiol. </li><li>2004 Sep </li></ul></li><li><p>Linearity of cortical receptive fields measured with natural <strong>sounds</strong>.</p><ul><li>Machens CK, Wehr MS, Zador AM.</li><li>J Neurosci. </li><li>2004 Feb 4</li></ul></li><li><p>Balanced inhibition underlies tuning and sharpens spike timing in <strong>auditory</strong> cortex.</p><ul><li>Wehr M, Zador AM.</li><li>Nature. </li><li>2003 Nov 27</li></ul></li><li><p>Binary spiking in <strong>auditory</strong> cortex.</p><ul><li>DeWeese MR, Wehr M, Zador AM.</li><li>J Neurosci. </li><li>2003 Aug 27</li></ul></li></ul><h2 class="anchor anchorWithStickyNavbar_y2LR" id="decision"><strong>decision</strong><a class="hash-link" href="#decision" title="Direct link to heading">â€‹</a></h2><ul><li><p>Standardized and reproducible measurement of <strong>decision</strong>-making in mice.</p><ul><li>International Brain Laboratory, Aguillon-Rodriguez V, Angelaki D, Bayer H, Bonacchi N, Carandini M, Cazettes F, Chapuis G, Churchland AK, Dan Y, Dewitt E, Faulkner M, Forrest H, Haetzel L, HÃ¤usser M, Hofer SB, Hu F, Khanal A, Krasniak C, Laranjeira I, Mainen ZF, Meijer G, Miska NJ, Mrsic-Flogel TD, Murakami M, Noel JP, Pan-Vazquez A, Rossant C, Sanders J, Socha K, Terry R, Urai AE, Vergara H, Wells M, Wilson CJ, Witten IB, Wool LE, Zador AM.</li><li>Elife. </li><li>2021 May 20</li></ul></li><li><p>Mice and rats achieve similar levels of performance in an adaptive <strong>decision</strong>-making task.</p><ul><li>Jaramillo S, Zador AM.</li><li>Front Syst Neurosci. </li><li>2014 Sep 18</li></ul></li></ul><h2 class="anchor anchorWithStickyNavbar_y2LR" id="computation"><strong>computation</strong><a class="hash-link" href="#computation" title="Direct link to heading">â€‹</a></h2><ul><li><p>Synaptic connectivity and <strong>computation</strong>.</p><ul><li>Zador AM.</li><li>Nat Neurosci. </li><li>2001 Dec</li></ul></li><li><p>The basic unit of <strong>computation</strong>.</p><ul><li>Zador AM.</li><li>Nat Neurosci. </li><li>2000 Nov</li></ul></li><li><p>Dynamic stochastic synapses as <strong>computation</strong>al units.</p><ul><li>Maass W, Zador AM.</li><li>Neural Comput. </li><li>1999 May 15</li></ul></li></ul><h2 class="anchor anchorWithStickyNavbar_y2LR" id="others">others<a class="hash-link" href="#others" title="Direct link to heading">â€‹</a></h2><ul><li><p>Identification of a brainstem locus that inhibits tumor necrosis factor.</p><ul><li>Kressel AM, Tsaava T, Levine YA, Chang EH, Addorisio ME, Chang Q, Burbach BJ, Carnevale D, Lembo G, Zador AM, Andersson U, Pavlov VA, Chavan SS, Tracey KJ.</li><li>Proc Natl Acad Sci U S A. </li><li>2020 Nov 24</li></ul></li><li><p>Differences in sensitivity to neural timing among cortical areas.</p><ul><li>Yang Y, Zador AM.</li><li>J Neurosci. </li><li>2012 Oct 24</li></ul></li><li><p>PINP: a new method of <strong>tagging</strong> neuronal populations for identification during in vivo electrophysiological recording.</p><ul><li>Lima SQ, HromÃ¡dka T, Znamenskiy P, Zador AM.</li><li>PLoS One. </li><li>2009 Jul 7</li></ul></li><li><p>Correlated connectivity and the distribution of firing rates in the neocortex.</p><ul><li>Koulakov AA, HromÃ¡dka T, Zador AM.</li><li>J Neurosci. </li><li>2009 Mar 25</li></ul></li><li><p>Sparse representations for the cocktail party problem.</p><ul><li>Asari H, Pearlmutter BA, Zador AM.</li><li>J Neurosci. </li><li>2006 Jul 12</li></ul></li><li><p>Natural scene statistics at the centre of gaze.</p><ul><li>Reinagel P, Zador AM.</li><li>Network. </li><li>1999 Nov</li></ul></li><li><p>Input synchrony and the irregular firing of cortical neurons.</p><ul><li>Stevens CF, Zador AM.</li><li>Nat Neurosci. </li><li>1998 Jul</li></ul></li><li><p>Efficient discrimination of temporal patterns by motion-sensitive neurons in primate visual cortex.</p><ul><li>Buracas GT, Zador AM, DeWeese MR, Albright TD.</li><li>Neuron. </li><li>1998 May</li></ul></li><li><p>Dynamic synapses in the cortex.</p><ul><li>Zador AM, Dobrunz LE.</li><li>Neuron. </li><li>1997 Jul</li></ul></li><li><p>Electrotonic architecture of hippocampal CA1 pyramidal neurons based on three-dimensional reconstructions.</p><ul><li>Mainen ZF, Carnevale NT, Zador AM, Claiborne BJ, Brown TH.</li><li>J Neurophysiol. </li><li>1996 Sep</li></ul></li><li><p>The morphoelectrotonic transform: a graphical approach to dendritic function.</p><ul><li>Zador AM, Agmon-Snir H, Segev I.</li><li>J Neurosci. </li><li>1995 Mar</li></ul></li></ul></div></div></details></div><footer class="theme-doc-footer docusaurus-mt-lg"><div class="theme-doc-footer-edit-meta-row row"><div class="col"><a href="https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/docs/1-Sequencing the Connectome/1-Papers.md" target="_blank" rel="noreferrer noopener" class="theme-edit-this-page"><svg fill="currentColor" height="20" width="20" viewBox="0 0 40 40" class="iconEdit_mS5F" aria-hidden="true"><g><path d="m34.5 11.7l-3 3.1-6.3-6.3 3.1-3q0.5-0.5 1.2-0.5t1.1 0.5l3.9 3.9q0.5 0.4 0.5 1.1t-0.5 1.2z m-29.5 17.1l18.4-18.5 6.3 6.3-18.4 18.4h-6.3v-6.2z"></path></g></svg>Edit this page</a></div><div class="col lastUpdated_mt2f"></div></div></footer></article><nav class="pagination-nav docusaurus-mt-lg" aria-label="Docs pages navigation"><div class="pagination-nav__item"></div><div class="pagination-nav__item pagination-nav__item--next"><a class="pagination-nav__link" href="/NeuroNotes/docs/Sequencing the Connectome/2012 Sequencing the Connectome"><div class="pagination-nav__sublabel">Next</div><div class="pagination-nav__label">2012 Sequencing the Connectome</div></a></div></nav></div></div><div class="col col--3"><div class="tableOfContents_vrFS thin-scrollbar theme-doc-toc-desktop"><ul class="table-of-contents table-of-contents__left-border"><li><a href="#sequencing-the-connectome" class="table-of-contents__link toc-highlight"><strong>Sequencing</strong> the connectome</a><ul><li><a href="#neuron-neuron-synapse-connectome" class="table-of-contents__link toc-highlight">Neuron-neuron synapse (connectome)</a></li><li><a href="#single-neuron-projection-projectome" class="table-of-contents__link toc-highlight">Single neuron projection (projectome)</a></li><li><a href="#toolbox" class="table-of-contents__link toc-highlight">Toolbox</a></li></ul></li><li><a href="#learning" class="table-of-contents__link toc-highlight">learning</a></li><li><a href="#auditory" class="table-of-contents__link toc-highlight"><strong>auditory</strong></a></li><li><a href="#decision" class="table-of-contents__link toc-highlight"><strong>decision</strong></a></li><li><a href="#computation" class="table-of-contents__link toc-highlight"><strong>computation</strong></a></li><li><a href="#others" class="table-of-contents__link toc-highlight">others</a></li></ul></div></div></div></div></main></div></div><footer class="footer footer--dark"><div class="container container-fluid"><div class="footer__bottom text--center"><div class="footer__copyright">Copyright Â© 2022 NeuroNotes, Inc. Built with Docusaurus.</div></div></div></footer></div>
<script src="/NeuroNotes/assets/js/runtime~main.9fa0cc15.js"></script>
<script src="/NeuroNotes/assets/js/main.288aa2cb.js"></script>
</body>
</html>